src/installing-workflow/configuring-registry.md

```diff
@@ -17,7 +17,7 @@ When using a private registry the docker images are no longer pulled by Deis Wor
 Deis Workflow currently supports
   1. Google Container Registry([gcr][gcr]).
   2. EC2 Container Registry([ecr][ecr]).
-  3. off-cluster storage providers like dockerhub, quay.io, etc., or self hosted docker registry.
+  3. off-cluster : Any provider which provides long lived auth(username/password) like [Azure Container Registry][acr], [dockerhub][dockerhub], [quay.io][quay], etc., or self hosted docker registry.
```

Comment:
And maybe it's not necessary to include `off-cluster:` here, since that's already implied.